common/rpc/encryption/certificate.go

```diff
@@ -0,0 +1,126 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
```

Comment:
Let's keep Uber everywhere in headers. In case if some legacy code is copied to new files.

Reply:
Confirmed with @samarabbas we will dual license everywhere